jwst/rscd/tests/test_rscd.py

```diff
@@ -0,0 +1,101 @@
+import numpy as np
+
+from jwst.datamodels import RampModel
+from jwst.datamodels import dqflags
+from jwst.rscd.rscd_sub import correction_skip_groups
+
+
+def test_rscd_baseline_set_groupdq():
+    """
+    For a 2 integration exposure, test if the rscd code sets the groupdq flag on
+    the n groups to 'do_not_use' for the 2nd integrations and did not change the
+    groupdq flags in the 1st integration
+    """
+
+    # size of integration
+    ngroups = 10
+    xsize = 1032
```

Comment:
Do you really need x/y dimensions this big just to test that flags were set for a few pixels? I would think something like 10x10 would be plenty and would make the test run a lot faster.

Reply:
You are correct.  Changed to 10 x 10.